### PR TITLE
Update test readme

### PR DESCRIPTION
**Pull request by frederikrothenberger**
The test readme has gone out of date.
Changes made:
* Fix flags required to build test flavour
* Include building the archive
* Include downloading the latest archive release in instructions

### CODE REVIEW

File: src/internet_identity/tests/README.md [hunk 1]
```diff
@@ -4,11 +4,17 @@ This is a test suite for the Internet Identity canister. To run the tests:
 
 ``` bash
 # Make sure II is built with the "test" flavor
-II_DUMMY_CAPTHA=1 ./scripts/build
+II_FETCH_ROOT_KEY=1 II_DUMMY_CAPTCHA=1 ./scripts/build
```
Comment (nmattia) [inline src/internet_identity/tests/README.md:7]: out of curiosity, why do these tests need II_FETCH_ROOT_KEY?

Comment (frederikrothenberger) [reply src/internet_identity/tests/README.md:7]: Actually, they don't. I added the flag to be consistent with the description in the main README (and the comment here).

But I can also revert that, but then I would not call it the `test` flavor here.

Comment (nmattia) [reply src/internet_identity/tests/README.md:7]: just curious :) thanks!

**Review by przydatek — APPROVED** [review]
LGTM, thanks!

**Pull request merged**